config/routes.rb

```diff
@@ -28,6 +28,8 @@
   root to: 'teachers/homes#show'
 
   # API
+  API_VERSIONS = [1].freeze
```

Comment:
I think this needs to be in an initializer as I'm getting constant redefined warning when I make a second request.